.github/workflows/build.yml

```diff
@@ -1,29 +1,32 @@
-name: Build and valiate SDK package
+name: Build and validate generated code
 
 on:
   workflow_dispatch:
-  workflow_call:
   push:
     branches: [ main ]
   pull_request:
     branches: [ main, dev ]
-
-permissions:
-  contents: read
+  workflow_call:
 
 jobs:
-  validate:
-    name: Validate code accuracy
```

Comment:
If you remove the name property here, the required check is no longer able to find the status to display.